addons/mail/static/src/core/public_web/user_status_panel.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.UserStatusPanel">
+        <div class="py-2 mt-auto position-sticky bottom-0 o-mail-discussSidebarBgColor" t-att-class="{ 'px-2': !store.discuss.isSidebarCompact }">
```

Comment:
Currently this looks too much like a channel item. It needs better distinction, e.g. with border above & some background